pandas/tests/io/formats/test_to_csv.py

```diff
@@ -561,3 +561,20 @@ def test_to_csv_compression(self, compression_only,
             result = pd.read_csv(path, index_col=0,
                                  compression=read_compression)
             tm.assert_frame_equal(result, df)
+
+
+def test_csv_formatter_line_terminator_default(monkeypatch):
```

Comment:
rather than this, can you just modify the appropriate tests to pass in ['\n', os.linesep, None] as parameters (we might already have this)

Reply:
> modify the appropriate tests to pass

@jreback There isn't an existing test of the `CSVFormatter` directly. 

The intent of the PR is to ensure that if no `line_terminator` is specified, the default is `os.linesep`. Currently, `CSVFormatter` must be called with `line_terminator=None` or else `\n` will be used. 

When I opened this PR, I was looking at `v0.24.3`, so I didn't realize the bug had been fixed in #21406 by calling the `CSVFormatter` class with `line_terminator=None`. I thought I might as well continue the PR so future devs don't have to remember to call `CSVFormatter(line_terminator=None)` if they expect the platform specific `os.linesep` behavior. Since most devs use `\n` platforms, it would be easy to miss this.

Reply:
i don't think this test is necessary and slightly confusing, can you remove.